data-requirements/fqm-e-dr.ts

```diff
@@ -0,0 +1,61 @@
+import fs from 'fs';
+import path from 'path';
+import { Calculator } from '../src';
+
+const RESET = '\x1b[0m';
+const FG_YELLOW = '\x1b[33m';
+const FG_GREEN = '\x1b[32m';
+
+const JAN_2024_CONNECTATHON_BASE_PATH = path.join(__dirname, './jan-2024-connectathon');
+
+/**
+ * The purpose of this function is to go through all of the measure bundles in the jan-2024-connectathon
+ * directory, calculate their data-requirements, and output their data-requirements to a JSON file
+ * corresponding to the name of the measure in the fqm-e-dr directory
+ */
+async function main() {
+  // if the fqm-e-dr directory already exists, remove it and its contents
+  if (fs.existsSync('./fqm-e-dr')) {
+    fs.rmSync('./fqm-e-dr', { recursive: true });
+  }
+
+  // create new fqm-e-dr directory within the data-requirements directory
+  fs.mkdirSync('./fqm-e-dr');
+
+  // get all of the file names (short and fullPath) from the jan-2024-connectathon directory
+  const allBundles = fs
+    .readdirSync(JAN_2024_CONNECTATHON_BASE_PATH)
+    .filter(f => f !== '.gitkeep')
```

Comment:
```suggestion
    .filter(f => !f.startsWith('.'))
```

Reply:
heh- came here to say this. Had the script get hung up on .ds_store